WordPress/gutenberg
# Site Editor: Add Post Taxonomies panel

Related #52632 

## What?

This PR continues the work on the great unification between post and site editors. This PR moves the PostTaxonomies panel from the edit-post package to the editor package and reuse the same panel in the site editor as well. This means that this brings supports to custom taxonomies and things like that to the site editor, similar to the post editor. 

## Testing instructions

 - Check that you can set tags and categories for posts.
 - Check that you can set tags for synced patterns in the site editor.
